frontend/src/routers/AppRouter/AppRouter.tsx

```diff
@@ -17,88 +17,121 @@ import SwitchWorkspace from '@pages/SwitchWorkspace/SwitchWorkspace'
 import InternalRouter from '@routers/InternalRouter/InternalRouter'
 import { DefaultWorkspaceRouter } from '@routers/OrgRouter/DefaultWorkspaceRouter'
```

Comment:
sessions and errors are always showing that no sessions exist?

<img width="426" alt="image" src="https://user-images.githubusercontent.com/1351531/218532019-fdf98ea6-8f2d-4a70-b8d9-dcb18dc1d235.png">

Reply:
This only replaces the info box saying that no session/error card is selected. Still possible to open the left panel